scripts/similarity_query.py

```diff
@@ -21,6 +29,25 @@ def parse_args(args):
 
 
 def main(args):
+
+    model_file_name = f"{similarity.model_name_to_class[args.algorithm].__name__.lower()}.similaritymodel"
+
+    if not os.path.exists(model_file_name):
+        logger.info(f"{model_file_name} does not exist. Downloading the model....")
+        try:
+            download_check_etag(
+                f"https://index.taskcluster.net/v1/task/project.relman.bugbug.train_similarity.latest/artifacts/public/{model_file_name}.zst",
```

Comment:
Nit: put `https://index.taskcluster.net/v1/task/project.relman.bugbug.train_similarity.latest/artifacts/public/{}.zst` in a URL variable at the top, and then here do URL.format(model_file_name)